crispy_tailwind/templates/tailwind/field.html

```diff
@@ -13,21 +13,27 @@
     {% endif %}
 
     {# if field has a special template then use this #}
-    {% if field|is_radioselect %}
+    {% if field|is_select %}
+        <div class="{% if field_class %} {{ field_class }}{% endif %}"{% if flat_attrs %} {{ flat_attrs|safe }}{% endif %}>
+
+        {% include 'tailwind/layout/select.html' %}
+    {% elif field|is_radioselect %}
         <div class="{% if field_class %} {{ field_class }}{% endif %}"{% if flat_attrs %} {{ flat_attrs|safe }}{% endif %}>
 
         {% include 'tailwind/layout/radioselect.html' %}
     {% elif field|is_checkboxselectmultiple %}
 
         <div class="{% if field_class %} {{ field_class }}{% endif %}"{% if flat_attrs %} {{ flat_attrs|safe }}{% endif %}>
-            {% include 'tailwind/layout/checkboxselectmultiple.html' %}
-            {% else %}
+        {% include 'tailwind/layout/checkboxselectmultiple.html' %}
+    {% else %}
 
-            {# otherwise use django rendering with additional classes added #}
-            {% tailwind_field field %}
+        {# otherwise use django rendering with additional classes added #}
+        {% tailwind_field field %}
+    {% endif %}
 
-            {% include 'tailwind/layout/help_text_and_errors.html' %}
+    {% include 'tailwind/layout/help_text_and_errors.html' %}
+
+    </{% if tag %}{{ tag }}{% else %}div{% endif %}>
```

Comment:
I'll need to take a closer look at this section here. I _think_ I had the closing `</div>` the wrong side of one the the `{% endif %} tags. My attempt at checkbox/radio then had an additional `</div>` to compensate for this error.

Reply:
Yeah, that was it. I moved it out one step and reformatted for readability, so this should be correct!